src/utils/tracer.ts

```diff
@@ -1,7 +1,13 @@
 import tracer from "dd-trace"
 
+import { config } from "../config/config"
+
+const env = config.get("dataDog.env")
+
 tracer.init({
-  logInjection: true, // https://docs.datadoghq.com/tracing/other_telemetry/connect_logs_and_traces/nodejs/
+  // NOTE: We require that the `logInjection` be dynamically set
+  // as otherwise it'll inject the dd object on local
+  logInjection: env !== "local", // https://docs.datadoghq.com/tracing/other_telemetry/connect_logs_and_traces/nodejs/
```

Comment:
Hmm, why do we have two separate envs for dd ah? like why cant we just use node_env here?

Reply:
we need 3 env vars for datadog [unified service tagging](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/). since this is a datadog specific config for env, i opted to use this rather than node env.